### DISCUSSION TITLE
Line number is only displayed for active line

### DISCUSSION THREAD

**Discussion by pkropachev** [LazyVim/LazyVim#3715] [Q&A] [answered]
Hi!

I faced with problem on nightly build of nvim. Line number is only displayed for active line. In the same time all is ok for v0.10.0.
Has somebody else faced with such problem?

![image](https://github.com/LazyVim/LazyVim/assets/34160907/01d504fc-c0e1-48bc-8be8-e795f9353a66)

**Comment by dpetka2001** [comment, marked as answer]
This is due to [this change in Neovim nightly](https://github.com/neovim/neovim/pull/29357). 

@folke I managed to overcome this by changing the instances of `%r` to `%l` locally (since `%l` is supposed to handle everything and `%r` is removed), but am unsure how that really works and don't know if maybe what I did locally has some other side-effects. So, leaving it to you to verify if this is the proper solution or something additional is needed as well. Sorry I can't be of further help.

PS: I imagine it would have to be guarded with something like `vim.fn.has("nvim-0.11")`, so it doesn't affect the stable version. Don't know if something can be taken into consideration about older nightlies.

**Reply by dpetka2001** [reply]
I ended up submitting a PR after all. Please feel free to disregard it if you think a more appropriate way of handling this is possible.
